cluster/src/main/java/com/hartwig/pipeline/metadata/LinkFileToSample.java

```diff
@@ -0,0 +1,17 @@
+package com.hartwig.pipeline.metadata;
+
+import com.hartwig.pipeline.sbpapi.FileResponse;
+import com.hartwig.pipeline.sbpapi.SbpRestApi;
+
+public class LinkFileToSample implements ApiFileOperation {
+    private final String sampleId;
+
+    public LinkFileToSample(String sampleId) {
+        this.sampleId = sampleId;
+    }
+
+    @Override
+    public void apply(final SbpRestApi api, final FileResponse fileResponse) {
```

Comment:
They also don't do much... maybe you can get away with just a BiFunction where you need the abstraction?